de-lib/src/main/java/org/iplantc/de/analysis/client/presenter/AnalysesPresenterImpl.java

```diff
@@ -431,22 +429,15 @@ public void onShowAllSelected() {
     @Override
     public void onShareAnalysisSelected(ShareAnalysisSelected event) {
         List<Analysis> selected = event.getAnalysisList();
-
-        AnalysisSharingViewImpl sharingView = new AnalysisSharingViewImpl();
-        final AnalysisSharingPresenter sharingPresenter = new AnalysisSharingPresenter(analysisService,
-                                                                                 selected,
-                                                                                 sharingView,
-                                                                                 collaboratorsUtil);
-       aSharingDialogProvider.get(new AsyncCallback<AnalysisSharingDialog>() {
+        aSharingDialogProvider.get(new AsyncCallback<AnalysisSharingDialog>() {
             @Override
             public void onFailure(Throwable caught) {
-
+                ErrorHandler.post(caught);
```

Comment:
Yeah, it's more habit than anything else 😄   There are probably times where I leave it blank, and then circle back around thinking, "Oops, I missed this."